src/windows/GCRealTimeMon.sln

```diff
@@ -1,12 +1,17 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio Version 16
-VisualStudioVersion = 16.0.31729.503
+# Visual Studio Version 17
+VisualStudioVersion = 17.0.31825.309
 MinimumVisualStudioVersion = 10.0.40219.1
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "GCRealTimeMon", "GCRealTimeMon.csproj", "{C816D5D9-38DF-44BC-8AA9-0059DD3ABD55}"
 EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "GCRealTimeMon.UnitTests", "..\..\tests\GCRealTimeMon.UnitTests\GCRealTimeMon.UnitTests.csproj", "{DAB632C8-4F62-4D1B-8209-F4D82C6A1990}"
+EndProject
 Global
 	GlobalSection(SolutionConfigurationPlatforms) = preSolution
+		Debug|Any CPU = Debug|Any CPU
+		Debug|x64 = Debug|x64
+		Release|Any CPU = Release|Any CPU
```

Comment:
are these intentional? I deleted those lines so when you build you would automatically build release x64. if you are developing this yourself, you can always change the config yourself.

Reply:
Ah, this was unintentional. Needed the Debug mode for the unit tests otherwise debugging them was challenging. I'll make sure to remove these. Thanks!